### DISCUSSION TITLE
How does one create a manifest for a msi?

### DISCUSSION THREAD

**Discussion by grafst** [ScoopInstaller/Main#4742] [Q&A]
I am trying to build a manifest for: https://www.arqbackup.com/download/arqbackup/Arq7.msi
The Wiki says to just extract it. However it seems there are some dependencies missing and so on.
Does anyone have any experience where to look in the msi for this stuff?

**Comment by rashil2000** [comment]
Which dependencies are missing? Could you paste your manifest draft here?
